README.md

```diff
@@ -69,6 +70,9 @@ A collection of useful links discovered through the work on [Weekly Robotics](ht
 
 
 ## Libraries and Frameworks
+ * [Rerun](rerun.io) - An SDK for logging computer vision and robotics data paired with a visualizer for exploring that data over time. It lets you debug and understand the internal state and data of your systems with minimal code.
```

Comment:
Usually, I don't feature companies here. Could you instead link to their repo https://github.com/rerun-io/rerun ? For each of the entries. Could you append `Licence: {{PROJECT_LICENCE}}` for each of the projects? It's probably tricky for Turtlebot but the rest should have quite clear licensing I hope

Reply:
Noted on the "no companies" and ***facepalm*** totally missed the Licences. Done!